srcpkgs/rockpro64-base/template

```diff
@@ -0,0 +1,11 @@
+# Template file for 'rockpro64-base'
+pkgname=rockpro64-base
+version=1
+revision=1
+archs="aarch64*"
+build_style=meta
+depends="rockpro64-uboot u-boot-menu dracut"
```

Comment:
doesn't this need a kernel? from what I see in mkplatformfs this would not create an image or PLATFORMFS with a kernel